app/views/cash_account_no_transactions_v2.scala.html

```diff
@@ -44,9 +44,12 @@
         id = Some("cash-account-authorise-an-agent")
     )
 
-    @p(
-        message = "cf.cash-account.top-up.guidance",
-        id = Some("cash-account-authorise-agent-guidance")
+    @link(
+        linkMessageKey = "cf.cash-account.top-up.guidance.text.link",
+        location = "https://www.gov.uk/guidance/paying-into-your-cash-account-for-cds-declarations",
```

Comment:
Please avoid hard coding urls, lets utilise the `application.conf` and `AppConfig`